linkerd2-proxy-init.yaml

```diff
@@ -0,0 +1,75 @@
+package:
+  name: linkerd2-proxy-init
+  version: 2.4.1
+  epoch: 0
+  description: "Init container that sets up the iptables rules to forward traffic into the Linkerd2 sidecar proxy"
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ip6tables
+      - iptables
+      - libcap
+      - libcap-utils
+  scriptlets:
+    post-install: |
+      #!/bin/sh
+      setcap cap_net_raw,cap_net_admin+eip /sbin/xtables-legacy-multi
+      setcap cap_net_raw,cap_net_admin+eip /sbin/xtables-nft-multi
```

Comment:
These can be dropped right? Since they won't be run in an apko image and the iptables package has the capabilities set?